spec/features/cart/user_can_add_items_to_cart_spec.rb

```diff
@@ -11,59 +11,81 @@
   end
 
   scenario "the cart counter says 1 after adding the first item" do
-    within "#item_#{pants.id}" do
-      click_on "Add to cart"
-    end
+    add_one_item_to_cart(pants)
 
```

Comment:
Upon discussion with team - this is a helper method that is being tested in the cart_model_spec

One additional test needed to clarify this method that lives in feature_helper

# with in item
#     expect page to have add to cart button
#end